### ISSUE TITLE
витоки

### ISSUE THREAD

**Issue by apsen-github** [brown-uk/dict_uk#370]
Слово "витоки" не розпізнається в hunspell.

![image](https://github.com/brown-uk/dict_uk/assets/115335511/b3728aac-f8e2-4985-8742-ddae7e1e69cc)

**Comment by arysin** [comment]
Так, ["витоки" є нерекомендованим словом](https://github.com/brown-uk/dict_uk/blob/master/data/dict/twisters.lst#L896) - рекомендований варіант джерела.
https://r2u.org.ua/s?w=%D0%B8%D1%81%D1%82%D0%BE%D0%BA%D0%B8&scope=all&dicts=all&highlight=on
https://e2u.org.ua/s?w=sources&dicts=all&highlight=on&filter_lines=on

**Comment by apsen-github** [comment]
Інше значення:
[https://e2u.org.ua/s?w=leak&dicts=all](url)

**Comment by arysin** [comment]
Ви маєте на увазі як множина від «витік»??

**Comment by apsen-github** [comment]
Так.

**Comment by arysin** [comment]
З теоретичного боку така множина правильна, але з практичного, якщо ми додамо «витоки», то це в 99% буде маскувати покруч. Підозрюю кращий варіант тут - додати цю форму в локальний словник.

**Comment by arysin** [comment]
Додано
